scripts/setup-worktree.test.sh

```
@@ -0,0 +1,298 @@
+teardown
+
+# --------------------------------------------------
+# Test 10: Idempotent_RunTwice_WorktreeExists
+# --------------------------------------------------
+setup
+# First run creates worktree
+PATH="$MOCK_BIN:$PATH" bash "$SCRIPT_UNDER_TEST" \
+    --repo-root "$REPO_DIR" \
+    --task-id "task-010" \
+    --task-name "idempotent" \
+    --skip-tests 2>&1 >/dev/null || true
```

Shellcheck (0.11.0)

[warning] To redirect stdout+stderr, 2>&1 must be last (or use '{ cmd > file; } 2>&1' to clarify).

(SC2069)